app/uk/gov/hmrc/apiplatformdeskpro/connector/DeskproConnector.scala

```diff
@@ -101,5 +112,24 @@ class DeskproConnector @Inject() (http: HttpClientV2, config: AppConfig, metrics
       .execute[DeskproOrganisationWrapperResponse]
   }
 
+  def getOrganisationsForPersonEmail(email: LaxEmailAddress)(implicit hc: HeaderCarrier): Future[DeskproLinkedOrganisationWrapper] = {
+    metrics.record(api) {
+      http
+        .post(url"${requestUrl(s"/api/v2/people")}")
+        .withProxy
+        .setHeader(AUTHORIZATION -> config.deskproApiKey)
+        .setHeader(DESKPRO_QUERY_HEADER -> DESKPRO_QUERY_MODE_ON)
+        .withBody(Json.parse(
+          s"""
```

Comment:
Personally I'd prefer to have a request model here rather than parsing a string.

Reply:
Ok, that's added